src/clr-angular/popover/tooltip/tooltip-content.ts

```diff
@@ -19,22 +20,25 @@ const SIZES: string[] = ['xs', 'sm', 'md', 'lg'];
     `,
   host: {
     '[class.tooltip-content]': 'true',
-    // I'm giving up on animation, they did not work before and will not work now.
-    // Too many conflicts with Clarity UI.
     '[style.opacity]': '1',
+    role: 'tooltip',
+    '[id]': 'tooltipId',
```

Comment:
Issue here is that it will clobber someone's custom ID. We should still allow them to be set by the dev.

Reply:
It seems like we still need a tooltip ID service here. In the custom case it's needed to communicate that there is a custom ID on the content and communicates that to the trigger.  Does that sound right?

Reply:
Could consider using the existing `AriaService` for this. I think it isn't used widely within our library but I do think it's a good fit given that we have to establish relationship between two elements in a few places in our components.

Reply:
I looked into `AriaService` but ultimately stayed with the unique id provider. I left a comment should we see benefit in centralizing the id for establishing relationships in the future.